Monika After Story/game/script-story-events.rpy

```diff
@@ -269,6 +269,187 @@ label monika_changename:
+        if bday_no_restart_ev is not None:
+            bday_no_restart_ev.start_date = datetime.datetime.combine(mas_player_bday_curr, datetime.time(hour=19))
+            bday_no_restart_ev.end_date = mas_player_bday_curr + datetime.timedelta(days=1)
+            bday_no_restart_ev.conditional = (
+                "mas_isplayer_bday() "
+                "and not persistent._mas_player_bday_in_player_bday_mode "
+                "and not persistent._mas_player_bday_decor "
+                "and not mas_isO31() "
+                "and not mas_isD25()")
+            bday_no_restart_ev.action = EV_ACT_QUEUE
+            Event._verifyAndSetDatesEV(bday_no_restart_ev)
+    
+        bday_holiday_ev = mas_getEV('mas_player_bday_other_holiday')
+        if bday_holiday_ev is not None:
+            bday_holiday_ev.start_date = mas_player_bday_curr
+            bday_holiday_ev.end_date = mas_player_bday_curr + datetime.timedelta(days=1)
+            bday_holiday_ev.conditional = (
+                "mas_isplayer_bday() "
+                "and not persistent._mas_player_bday_in_player_bday_mode "
+                "and not persistent._mas_player_bday_decor "
+                "and "
+                "mas_isO31() or mas_isD25()")
+            bday_holiday_ev.action = EV_ACT_QUEUE
+            Event._verifyAndSetDatesEV(bday_holiday_ev)
+
+        persistent._mas_player_confirmed_bday = True
+
+    if old_bday is not None:
+        $ old_bday_not_none = old_bday.replace(year=mas_player_bday_curr.year)
+
+    if not mas_isplayer_bday() and old_bday_not_none == mas_player_bday_curr:
+        return
+
+    if mas_isplayer_bday():
+        if old_bday_not_none == mas_player_bday_curr:
+            if mas_isMoniNormal(higher=True):
+                m 3hub "Ahaha! So today {i}is{/i} your birthday!"
+                m 1tsu "I'm glad I was prepared, ehehe..."
+                m 3eka "Hold on just one moment, [player]..."
+                show monika 1dsc
+                pause 2.0
+                jump mas_player_bday_when_confirmed
+            elif mas_isMoniDis(higher=True):
+                $ persistent._mas_player_bday_decor = False
+                m 2eka "Ah, so today {i}is{/i} your birthday..."
+                m "Happy Birthday, [player]."
+                m 4eka "I hope you have a good day."
+            else:
+                $ persistent._mas_player_bday_decor = False
+            return
+        else:
+            $ persistent._mas_player_bday_decor = False
+            if mas_isMoniNormal(higher=True):
+                $ persistent._mas_player_bday_in_player_bday_mode = True
+                m 1wuo "Oh...{w=1}Oh!"
+                m 3sub "Today's your birthday!"
+                m 3hub "Happy Birthday, [player]!"
+                m 1rksdla "I wish I had known earlier so I could've prepared something."
+                m 1eka "But I can at least do this..."
+                call mas_player_bday_moni_sings
+            elif mas_isMoniDis(higher=True):
```

Comment:
```suggestion
                m 1hub "Ahaha! It's not much but it's something!"
                m 3hua "I promise next year we'll do something extra special, [player]!"
            elif mas_isMoniDis(higher=True):
                m 2eka "Oh, so today's your birthday..."
```

Reply:
oops good catch.  had just altered those conditions before opening the PR and put that line in the wrong spot.  thanks! just turned my computer off, will fix that tomorrow